js/src/SmartConnect/api.md

```diff
@@ -6,32 +6,35 @@ a sessionURL is already provided in the configuration
 it will establish a direct WebSocket connection using
 Autobahn.
 
-## SmartConnect.newInstance({ config }) 
+## SmartConnect.newInstance({ config })
 
 Create an instance that will use the provided configuration to
 connect itself to a server either by requesting a new remote
 process or by trying to directly connecting to it as a fallback.
 
-## connect() 
+## connect()
 
 Trigger the connection request.
 
 ## onConnectionReady(callback) : subscription
 
 Register callback for when the connection became ready.
 
-## onConnectionClose(callback) : subscription
+## onConnectionClose(callback, err) : subscription
 
-Register callback for when the connection close.
+Register callback for when the connection close.  Err is a websocket event. If
+the server closes the connection after sending a close frame
+(https://datatracker.ietf.org/doc/html/rfc6455#section-5.5.1), the err object
+will have shape {code: number, reason: string}.
 
-## onConnectionError(callback) : subscription
+## onConnectionError(callback, err) : subscription
```

Comment:
err is passed to the callback. It is not part of the profile of the `onConnectionError`

Reply:
You are right. Rephrased the doc.